tests/integration/__init__.py

```diff
@@ -1112,22 +1133,6 @@ def __exit__(self, type, value, traceback):
             self.smaster_process.terminate()
         except AttributeError:
             pass
-        #salt.utils.process.clean_proc(self.sub_minion_process, wait_for_kill=50)
```

Comment:
Do we need this?

Reply:
It has been commented out for quite a while, hence me deleting it.
I'd say, no.

Reply:
This code is being submitted separately in https://github.com/saltstack/salt/pull/54263

Reply:
OMG, I didn't see that was getting removed. Even though it's clearly red. Guess that was before my coffee. ☕️

Reply:
:smile: